Sources/Application/SceneDelegate.swift

```diff
@@ -20,6 +20,13 @@ class SceneDelegate: UIResponder,
         return UserStorage(secureStoreService: secureStoreService,
                                     defaultsStore: UserDefaults.standard)
     }()
+    private lazy var openAccessUserStore = {
+        let secureStoreService = SecureStoreService(configuration: .init(id: .persistentSessionID,
+                                                                         accessControlLevel: .open,
+                                                                         localAuthStrings: LAContext().contextStrings))
+        return UserStorage(secureStoreService: secureStoreService,
+                                    defaultsStore: UserDefaults.standard)
+    }()
```

Comment:
Could this be part of `userStore`? Feels reasonably like something which could exist within that type 🙂